pkg/chunk/aws_storage_client.go

```diff
@@ -410,6 +412,11 @@ func (a dynamoDBRequestAdapter) Retryable() bool {
 	return *a.request.Retryable
 }
 
+type chunksPlusError struct {
```

Comment:
Obligatory type theory nerdery: this is more accurately `chunksTimesError`. I'm not actually suggesting you change this.